docs/source/auth/rest-api.rst

```diff
@@ -0,0 +1,501 @@
+
+.. _auth-rest-api:
+
+==============================
+MLflow Authentication REST API
+==============================
+
```

Comment:
Can we add a note and say this is an experimental feature?

Reply:
sure